types/datatables.net-buttons/index.d.ts

```diff
@@ -443,6 +443,15 @@ declare namespace DataTables {
         columnText?: FunctionButtonColvisColumnText;
 
         //#endregion ColVis
+
+        //#region Buttons
+
+        /**
+         * List of buttons to be created
+         */
+        buttons?: Array<(string|ButtonSettings)>;
```

Comment:
Should this include `FunctionButtom`, as in `ButtonsSections`?

Reply:
Good point, I missed it, thx